doc/examples/nonlinear_from_rest/run_simul_check_from_python.py

```diff
@@ -39,10 +40,42 @@
 
 parser.add_argument("-nx", type=int, default=12, help="number of x elements")
 parser.add_argument("-nz", type=int, default=12, help="number of z elements")
-parser.add_argument("--order", type=int, default=10, help="order")
+parser.add_argument("--order", type=int, default=10, help=" polynomial order")
 parser.add_argument("--dim", type=int, default=2, help="2d or 3d")
 parser.add_argument("--stretch-factor", type=float, default=0.0, help="stretch factor")
 
+parser.add_argument(
+    "--x-periodicity",
+    type=strtobool,
```

Comment:
`type=bool` should be fine.

Reply:
It was not fine, it takes it as `True` always. I'll check it again.

Reply:
You need to use https://docs.python.org/3/library/argparse.html#action

```python
parser = argparse.ArgumentParser()
parser.add_argument('--foo', action='store_true')
parser.add_argument('--bar', action='store_false')
parser.add_argument('--baz', action='store_false')
parser.parse_args('--foo --bar'.split())
```